py/strato/common/log/log2text.py

```diff
@@ -33,17 +37,22 @@
     "\033[2;36m",
 )
 COLOR_OFF = "\033[0;0m"
+LOG_CONFIG_FILE_PATH = "/etc/strato-log.conf"
+HIGHEST_PRIORITY = 0
 
 class Formatter:
     _COLORS = {logging.PROGRESS: CYAN, logging.ERROR: RED, logging.WARNING: YELLOW}
 
     converter = time.gmtime
 
     def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecondPrecision, noColors, localTime=False):
+        self.configFile = yaml.load(open(LOG_CONFIG_FILE_PATH, 'r').read())
```

Comment:
if the file is not there or incorrect format, you'll crash, no? 
Please handle the exception correctly. 
You might want to print a warning that you wont be able to parse all logs correctly and continue